openshift-knative-operator/pkg/serving/extension.go

```diff
@@ -116,6 +116,11 @@ func (e *extension) Reconcile(ctx context.Context, comp v1alpha1.KComponent) err
 			Type: "ConfigMap",
 		}
 	}
+
+	// Temporary fix for SRVKS-743
+	if ks.Spec.Ingress.Istio.Enabled {
+		common.Configure(&ks.Spec.CommonSpec, monitoring.ObservabilityCMName, monitoring.ObservabilityBackendKey, "none")
```

Comment:
let's make this `ConfigureIfUnset` just to leave the gate open for people who **really** want to play with it enabled explicitly.

Reply:
Ok good to know updating in a sec.